README.md

```diff
@@ -2,3 +2,61 @@
 
 [![PyPI version](https://badge.fury.io/py/vald-client-python.svg)](https://badge.fury.io/py/vald-client-python)
 
+A Python gRPC client library for [Vald](https://github.com/vdaas/vald).
+
+## Install
+
+Using pip,
+
+```sh
+pip install grpcio-tools
```

Comment:
If the users just want to use vald-client-python, they need not to install grpcio-tools.

Reply:
Thanks